### DISCUSSION TITLE
Question about the use of two Y-axis

### DISCUSSION THREAD

**Discussion by changellenger** [nhn/tui.chart#637] [Q&A]
안녕하세요
현재 아래 사진처럼 두 개의 y축을 사용하고 있습니다.

왼쪽 y 축의 요일별 인원은 하늘색의 신규 가입 수를 표현하고 싶고, 
오른쪽의 y축의 누적인원은 노란색의 누적 가입자 수 를 표현하려 하는데
두 데이터의 차이가 너무 커서 구분이 힘든 상황입니다.

![image](https://user-images.githubusercontent.com/51071820/112611424-26de3200-8e61-11eb-9fb1-195c5bf9ed15.png)

아래의 사진처럼 각각의 y 축에 따른 그래프를 그릴 수 있을까요?

![image](https://user-images.githubusercontent.com/51071820/112611682-7d4b7080-8e61-11eb-8e00-f55ec6a00ab4.png)

**Comment by jung-han** [comment]
@changellenger 

Single line chart does not currently provide this option. 
If you specify each `chartType` using the `lineArea` chart, it seems like you can represent the corresponding chart. How about this?

* https://github.com/nhn/tui.chart/blob/main/docs/ko/common-axes.md#secondary-y-axis
